controllers/release/release_suite_test.go

```diff
@@ -115,3 +118,26 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+// GetModuleCRDPath return the filesystem path after GOPATH/pkg/mod
+// for pseudo versioned modules.
+func GetModuleCRDPath(goModPath string, moduleName string) string {
+	var CRDPath string
+	readFile, err := os.Open(goModPath)
+	if err != nil {
+		panic(err)
+	}
+	scanner := bufio.NewScanner(readFile)
+	for scanner.Scan() {
+		re := regexp.MustCompile(".*" + moduleName + ".*")
+		match := re.Find([]byte(scanner.Text()))
+		if match != nil {
+			CRDPath = strings.Trim(string(match), "\t")
+			break
+		}
+	}
+	readFile.Close()
```

Comment:
You can use `defer readFile.Close()` so it's closed when you leave the function.

Reply:
fixed